docs/book/installation/upgrading.rst

```diff
@@ -4,53 +4,26 @@
 Upgrading
 =========
 
-Sylius is releasing new versions from time to time. Each release is supported with an `UPGRADE <https://github.com/Sylius/Sylius/blob/1.0/UPGRADE-1.0.md>`_ file, which is meant to help in the upgrading process,
-especially for the major versions, which can break the backwards compatibility.
+Sylius regularly releases new versions according to our :doc:`release process </contributing/organization/release-process>`.
```

Comment:
Shouldn't it be `release cycle`?

Reply:
It's called "release process" in our docs, so I'd stick to that for now.